### PR TITLE
Expose current time in `Broker`

### PR DESCRIPTION
**Pull request by Impelon**
I have another small suggested addition. This PR is less important to me than the other PRs made this week.
Instead of having to assume the current time (which I do now) or extracting it as the last value from one of the time series, it could be helpful to export this property, too.

Another (only indirectly related) aspect one may want to think about converting these times (or their type annotations) to a common representation, too. (Some kind of datetime object instead of a DatetimeLike.)

### CODE REVIEW

File: vessim/sil.py [hunk 1]
```diff
@@ -66,6 +68,10 @@ def set_event(self, category: str, value: Any) -> None:
             }
         )
 
+    @property
+    def current_time(self) -> DatetimeLike | None:
```
Comment (marvin-steinke) [inline vessim/sil.py:72]:
```suggestion
    def time(self) -> DatetimeLike | None:
```
Maybe cleaner?

Comment (Impelon) [reply vessim/sil.py:72]: Sure, why not :)

**Review by marvin-steinke — APPROVED** [review]

**Comment by Impelon** [comment]
Right, I didn't notice the merge conflicts, I fixed them now :)

**Comment by marvin-steinke** [comment]
We're currently discussing a redesign of the actor interface and are putting development of the SiL module and also the Broker as this will probably influence their interface as well.

**Comment by marvin-steinke** [comment]
I will check for compatibility with our new stuff and merge in the next couple of days :)

**Review by birnbaum — APPROVED** [review]

**Pull request closed without merging**